### PR TITLE
Remove SB14 code

### PR DESCRIPTION
**Pull request by madhurajayaraman**
b/355449840

### CODE REVIEW

**Comment by madhurajayaraman** [comment]
Updating SB max version to 17:
https://github.com/youtube/cobalt/pull/3884

Cleanup for SB14:
https://github.com/youtube/cobalt/pull/3899
https://github.com/youtube/cobalt/pull/3895
https://github.com/youtube/cobalt/pull/3897
https://github.com/youtube/cobalt/pull/3902

**Comment by andrewsavage1** [comment]
> Updating SB max version to 17: #3884
> 
> Cleanup for SB14: #3899 #3895 #3897 #3902

Why do you want the cleanups to be in separate PRs? They're all doing the same thing, no?

Also it looks like they cover most stuff, but I think you missed searching for `SB_API_VERSION == 14`

**Comment by madhurajayaraman** [comment]
@andrewsavage1 cleanups are in separate CLs since I didn't know how large the change would be, so broke it up into smaller CLs as I was making the changes. It's generally easier to review IMO. I did miss the == 14 changes, so I'll create a followup that removes those + has the minimum API version bumped up to 15.

**Comment by andrewsavage1** [comment]
> @andrewsavage1 cleanups are in separate CLs since I didn't know how large the change would be, so broke it up into smaller CLs as I was making the changes. It's generally easier to review IMO. I did miss the == 14 changes, so I'll create a followup that removes those + has the minimum API version bumped up to 15.

Okay, just add a reference to the new PR here once you have the number, that way github will create links to the others when they're all merged

**Review by andrewsavage1 — APPROVED** [review]

File: cobalt/media/base/sbplayer_pipeline.cc [hunk 1]
```diff
@@ -208,6 +202,7 @@ SbPlayerPipeline::SbPlayerPipeline(
```
Comment (xiaomings) [inline cobalt/media/base/sbplayer_pipeline.cc:202]: Shall we also remove the code above guarded by "#if SB_API_VERSION < 15"?

**Pull request merged**